test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -429,34 +464,54 @@ export default class PerfCompare extends Component {
                     continue;
                 }
 
+                // get the raw values for master nodes.
                 curMetricName = this.state.benchmarkRuns.benchmarkRunBaseline.parsedVariants[i].metrics[j].name;
+                let childBuildBaselineData = {};
```

Comment:
`handleGenerateTable` is used for both Jenkins and Perffarm so it would be better to move all Jenkins specific stuff to `handleGetJenkinsRuns`. We should also do some testing to make sure that we aren't breaking this tool for Perffarm.